miniapp/USERGUIDE.md

```diff
@@ -123,7 +123,7 @@ class MiniAppActivity : Activity(), CoroutineScope {
                     val miniAppInfo = MiniApp.instance().fetchInfo("MINI_APP_ID") // Or use `MiniApp.listMiniApp` if you want the whole list of Mini Apps
                     MiniApp.instance().create(miniAppInfo, miniAppMessageBridge)
                 }
-                val miniAppView = miniAppDisplay.getMiniAppView()
+                val miniAppView = miniAppDisplay.getMiniAppView(this)
```

Comment:
`this` here is `CoroutineScope` from `launch`. The correct context passing is `this@MiniAppActivity`.

Reply:
Oops, yeah you're right. I've updated this.